### DISCUSSION TITLE
no more e-mail from prints - test-mail arrives

### DISCUSSION THREAD

**Discussion by dermaxem** [berrystephenw/OctoText#116] [Q&A]
since a few weeks i have the problem, that no e-mails arrive anymore. the test-mail arrives, any other mail that was send weeks ago (start, Pause, finish a printjob) does not arrive. this worked well some weeks ago. the target-e-mail-adress is a "gmail.com", i checked it well, also the spam-Folder. no mails from the Printer there.

**Comment by berrystephenw** [comment]
That’s odd. Normally if the test email works you would be all set. Please turn on debug level logging for octotext and attach the log file from octoprint.

Stephen Berry
Hardware Engineering Manager
GSSI
40 Simon St
Nashua, NH 03060
________________________________
From: dermaxem ***@***.***>
Sent: Thursday, January 13, 2022 4:58:37 PM
To: berrystephenw/OctoText ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [berrystephenw/OctoText] no more e-mail from prints - test-mail arrives (Discussion #116)


since a few weeks i have the problem, that no e-mails arrive anymore. the test-mail arrives, any other mail that was send weeks ago (start, Pause, finish a printjob) does not arrive. this worked well some weeks ago. the target-e-mail-adress is a "gmail.com", i checked it well, also the spam-Folder. no mails from the Printer there.

**Reply by marioojedaf** [reply]
Hi, same issue for me. Only test emails will be sent, but no event-triggered emails come out.
I'm sending through an Outlook.com address, into a Gmail address.
Is there a known solution to this?
Thanks.

**Reply by berrystephenw** [reply]
I need logs to see if there is any issue with the plugin. My suspicion is that gmail has changed their spam policy but I can't verify that without more information.

**Comment by thenixhex311** [comment]
Experiencing the same issues to. Just found this plug-in yesterday...otherwise, this is an amazing feature to implement!

**Comment by maxu359** [comment]
So this happened to me too. Test arrives but no other mails... After I uninstalled some unused plugins, it started to work again. Can't say exactly which plug-in, because I uninstalled about 6 plugins.
But there was definitely a plug-in, which interrupted the mails.
Perhaps this helps

**Reply by berrystephenw** [reply]
Hi - this might help but I would need the name of the plugin's that you had installed/removed.

**Reply by maxu359** [reply]
Had to take a look in the logs.
I tried something a while ago, so I installed some plugins.
These are the names from the log, which I uninstalled:
toptemp
navbartemp
customControl
gcodesystemcommands
actioncommands

**Reply by berrystephenw** [reply]
awesome, that gives me a place to start. What version of Octotext are you running?

**Reply by maxu359** [reply]
OctoPrint 1.8.6
Python 3.7.3
OctoPi 0.18.0
Awesome Plug-in btw:)

**Reply by berrystephenw** [reply]
Hi max - I loaded all those plugin's and had no problem with test messages. I'll run a test print but it occurred to me that the action command plugin might have something to do with it depending on the settings in the plugin. Send me the settings you had?

**Reply by berrystephenw** [reply]
no problem with the test print.

**Reply by maxu359** [reply]
Im sorry, but I don't have the config anymore... 
I removed most plugins... Your plugin ist the most important one for me:)

**Comment by dermaxem** [comment]
Seems OK now,  bit I changed a lot are Marlin and Octoprint. thank you.kind regards.Mit freundlichen Grüßen,Matthias Lach
